benchmarks/graph_infra/arkouda-sort-cases.graph

```diff
@@ -0,0 +1,41 @@
+perfkeys: uniform int64 16-bit average rate =, uniform int64 32-bit average rate =, uniform int64 64-bit average rate =, uniform float64 (0, 1) average rate =
+graphkeys: 16-bit, 32-bit, 64-bit, float64
+files: sort-cases.dat
```

Comment:
`files` is a comma separated list of files with one entry per perfkey/graphkey. So you probably want this to be:

```
files: sort-cases.dat, sort-cases.dat, sort-cases.dat, sort-cases.dat
```

Alternatively you can use `repeat-files: sort-cases.dat`, but this feature isn't as stable in our testing infrastructure so I'd probably just go the more annoying route of duplicating filenames.

Reply:
Thanks! I will hopefully get a chance to update this week.

Reply:
I updated the file list, but I still can't seem to get the new graphs to display when I run locally. Let's come back to this next week. Happy Thanksgiving!

Reply:
Not sure if this is the issue or something else, but note that you can't just "open" graphs locally (because of javascript cross-site security issues.) To browse locally you need to start an http server. https://github.com/Bears-R-Us/arkouda/blob/master/benchmarks/graph_infra/README.md should have instructions for how to do that.

Reply:
Oh yeah, I've been running a local http server, but no luck. But now I'm wondering if it's somehow caching the old index or something. Anyway, you're off this week, right? You seriously do not have to worry about this right now :)

Reply:
Sounds good. 

Mostly off :) In for a bit today to finish up some paperwork I put off, and I was using this to procrastinate even more :)

Reply:
Ah, say n'more, say n'more. Happy to help :)